mmv1/products/vertexai/api.yaml

```diff
@@ -103,6 +103,238 @@ objects:
+              #     If machineSpec.accelerator_count is 0, the autoscaling will be based on CPU utilization metric only with default target value 60 if not explicitly set.  
+              #     For example, in the case of Online Prediction, if you want to override target CPU utilization to 80, you should set autoscalingMetricSpecs.metric_name to aiplatform.googleapis.com/prediction/online/cpu/utilization and autoscalingMetricSpecs.target to 80.
+          - !ruby/object:Api::Type::NestedObject
+            name: 'automaticResources'
+            description: |
+              A description of resources that to large degree are decided by Vertex AI, and require only a modest additional configuration.  
+              Cannot be used with `dedicatedResources`
+            properties:
+              - !ruby/object:Api::Type::Integer
+                name: 'minReplicaCount'
+                description: |
+                  Immutable. The minimum number of replicas this DeployedModel will be always deployed on. If traffic against it increases, it may dynamically be deployed onto more replicas up to `maxReplicaCount``, and as traffic decreases, some of these extra replicas may be freed. If the requested value is too large, the deployment will error.
+              - !ruby/object:Api::Type::Integer
+                name: 'maxReplicaCount'
+                description: |
+                  Immutable. The maximum number of replicas this DeployedModel may be deployed on when the traffic against it increases. If the requested value is too large, the deployment will error, but if deployment succeeds then the ability to scale the model to that many replicas is guaranteed (barring service outages). If traffic against the DeployedModel increases beyond what its replicas at maximum may handle, a portion of the traffic will be dropped. If this value is not provided, a no upper bound for scaling under heavy traffic will be assume, though Vertex AI may be unable to scale beyond certain replica number.
+      - !ruby/object:Api::Type::KeyValuePairs
+        name: 'trafficSplit'
+        description: |
+          A map from a DeployedModel's ID to the percentage of this Endpoint's traffic that should be forwarded to that DeployedModel.  
+          If a DeployedModel's ID is not listed in this map, then it receives no traffic.      
+          The traffic percentage values must add up to 100, or map must be empty if the Endpoint is to not accept any traffic at a moment.
+      - !ruby/object:Api::Type::String
+        name: 'etag'
+        description: Used to perform consistent read-modify-write updates. 
+        output: true
+      - !ruby/object:Api::Type::KeyValuePairs
+        name: 'labels'
+        description: |
+          A set of key/value label pairs to assign to this Endpoint.
+        
+      - !ruby/object:Api::Type::String
```

Comment:
We can drop `createTime` and `updateTime`, that'll make the change a little smaller and I don't feel they have much value for users.